backend/stakepoold/rpc/api.proto

```diff
@@ -169,3 +169,8 @@ message GetStakeInfoResponse {
 	double ProportionMissed = 15;
 	uint32 Expired = 16;
 }
+
+message SetMaxUsersRequest {
+	uint32 MaxUsers = 1;
+}
+message Empty{}
```

Comment:
@jholdstock 
What do you think about this? If we don't need a value, this can be reused, but it is different from the other code. I can change it to SetMaxUsersResponse.

Reply:
I would suggest using a dedicated `SetMaxUsersResponse`, even if it includes no fields right now. That will enable adding fields to the response in the future without incrementing the API major version number.

Reply:
Hadn't even thought about that. ok.